website/docs/d/fabric_storage_policy_vsphere.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "vra"
+page_title: "VMware vRealize Automation: fabric_storage_policy_vsphere"
+description: |-
+  Provides a data lookup for fabric vSphere storage policy.
+---
+
+# Data Source: fabric_storage_account_vSphere
+## Example Usages
+This is an example of how to lookup fabric vSphere storage policies.
+
+**Fabric vSphere storage policy by Id:**
+
+```hcl
+# Lookup fabric vSphere storage account using its Id
+data "fabric_storage_policy_vsphere" "this" {
+  id = var.fabric_storage_policy_vsphere_id
+}
+```
+
+**Fabric vSphere storage policy by filter query:**
+
+```hcl
+# Lookup fabric vSphere storage policy using its name
+data "fabric_storage_policy_vsphere" "this" {
+  filter = "name eq '${var.name}'"
+}
+```
+
+A fabric vSphere storage policy supports the following arguments:
+
+## Argument Reference
+* `filter` - Search criteria to narrow down the fabric vSphere storage policy. Only one of 'filter' or 'id' must be specified.
+
+* `id` - The id of the fabric vSphere storage policy. Only one of 'filter' or 'id' must be specified.
+
+## Attribute Reference
+* `cloud_account_ids` - Set of ids of the cloud accounts this entity belongs to.
+
+* `created_at` - Date when the entity was created. The date is in ISO 6801 and UTC.
+
+* `description` - A human-friendly description of the fabric vSphere storage account.
```

Comment:
`description` is not part of the schema. If the API supports it, either add the code to map the value or remove it here from docs.